client/my-sites/plugins/incompatible-plugins.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * This is a list of plugins not supported on WordPress.com Atomic sites.
+ *
+ * If this list is modified, please ensure that the support document also reflects
+ * any changes made:
+ *  - https://en.support.wordpress.com/incompatible-plugins
+ *
+ * Please keep this list alphabetized within the different categories!
+ */
+
+export const IncompatiblePlugins = [
```

Comment:
I think `IncompatiblePlugins` should probably be lower-cased.

Since we are conceptually treating this as a set, we can go ahead and make it a real [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set).

```javascript
const incompatiblePlugins = new Set( [
  // ...
] );
```

And since all the consumers of this module are simply checking whether something is included in the set, maybe we should export an `isIncompatiblePlugin()` function instead of the data structure we're using to represent the set.

Further, to avoid the awkwardness and confusion of using is-negative function (confusing when negating the result like `! isIncompatiblePlugin()`), maybe we should rename this module to `plugin-compatibility.js` and export an `isCompatiblePlugin()` function.